db/saveUserData.ts

```diff
@@ -0,0 +1,21 @@
+import { IUserDetails } from '../@types/types'
+import { db } from './getDbInstance'
+
+export default async (userData: IUserDetails) => {
+
```

Comment:
Do you think this code is easy to edit? How are you sure you are putting the parameters in the right order? Does the library offer a better solution?

Reply:
It wasn't. I refactored it and it's using named parameters.